Expensify/App
# Update "be the first person to comment" screen

@michaelhaxhiu @roryabraham
### Details
Updated the "Be the first person to comment!" screen to a new look here is the Figma [link](https://www.figma.com/file/rReWxiEhGDHIjajsE9XoJw/Start-a-chat?node-id=0%3A1).
As we discussed in the [issue](https://github.com/Expensify/App/issues/4678).

### Fixed Issues

$ https://github.com/Expensify/App/issues/4678

### Tests

Run app on any platform
Login with any account on any platform,
1. Click on the plus button add a new contact (email/user) or create a new group.
2. You will see the new and updated screen for "Be the first person to comment"
3. If you created a new chat with a single user it will show its time zone on top of the input otherwise will not.
4. Avatar(s) will show for the user in a group or single chat along with a description.
5. for private groups it will show a custom icon and custom description
please refer to [Figma](https://www.figma.com/file/rReWxiEhGDHIjajsE9XoJw/Start-a-chat?node-id=0%3A1) for more detail.



### QA Steps

For example:
1. Click on the plus button add a new contact (email/user) or create a new group.
2. You will see the new and updated screen for "Be the first person to comment"
3. If you created a new chat with a single user it will show its time zone on top of the input otherwise will not.
4. Avatar(s) will show for the user in a group or single chat along with a description.
5. for private groups it will show a custom icon and custom description
previously it was only "be the first to comment" text on the whole screen.


### Tested On

- [] Desktop
- [] iOS

### Screenshots
#### Web
<img width="1917" alt="Screenshot 2021-08-31 at 12 52 51 AM" src="https://user-images.githubusercontent.com/31244700/131397220-247c1d40-e104-4d11-bec3-97b2a7714323.png">

<img width="1918" alt="Screenshot 2021-08-31 at 12 52 37 AM" src="https://user-images.githubusercontent.com/31244700/131397260-96ac8a63-125f-4561-90da-7f462e33a81d.png">

#### Mobile Web
<img width="375" alt="Screenshot 2021-08-31 at 12 53 20 AM" src="https://user-images.githubusercontent.com/31244700/131397147-d285dbc6-a9c2-40a2-bb81-d0ca58cc2977.png">

<img width="373" alt="Screenshot 2021-08-31 at 12 53 33 AM" src="https://user-images.githubusercontent.com/31244700/131397166-1434f7cb-0e67-47e3-953c-8d16ca2d1c5e.png">

#### Desktop


#### iOS


#### Android
![Screenshot_2021-08-31-22-04-59-687_com expensify chat](https://user-images.githubusercontent.com/31244700/131546627-774c543a-9ec4-4bbf-a916-da6b24275bb3.jpg)
![Screenshot_2021-08-31-22-04-32-002_com expensify chat](https://user-images.githubusercontent.com/31244700/131546643-670f1ab0-c719-4229-8bb0-8b4378cba5e9.jpg)
